docs/source/how_to_guides/configure_cloud_storage_credentials.md

```diff
@@ -72,6 +72,20 @@ export MOSAICML_STREAMING_AWS_REQUESTER_PAYS='streaming-bucket,another-bucket'
 ```
 ````
 
+### Canned ACL
+Canned ACLs (Access Control Lists) are predefined sets of permissions in AWS S3 that you can apply to your objects. These can simplify the process of managing access to your S3 resources. Examples of canned ACLs include `private`, `public-read`, `public-read-write`, `authenticated-read`, etc. You can set a canned ACL for your S3 objects by using the `S3_CANNED_ACL` environment variable. This allows you to manage access permissions to your S3 resources in a simplified manner.
+
+````{tabs}
+```{code-tab} py
```

Comment:
just want to make sure that this displays correctly in the docs, because when I look at the preview on github the ```{code-tab} py line is visible, for example.

Reply:
It is correctly rendered. Checkout [this](https://mosaicml-docsite--512.com.readthedocs.build/projects/streaming/en/512/) link.